pkg/controller/node/status.go

```diff
@@ -103,6 +104,9 @@ func calculateStatus(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) mcfgv
 	}
 
 	var nodeDegraded bool
+	for _, m := range degradedMachines {
+		glog.Infof("Degraded Machine: %v and Degraded Reason: %v", m.Name, m.Annotations[constants.MachineConfigDaemonReasonAnnotationKey])
```

Comment:
out of curiosity was this for debugging purposes or do you want to add this extra log? I am not against it, but I thought we would have errored with degraded reason somewhere already

Reply:
I thought leaving this one in could be helpful for future issues with degraded machines. I could not find a part in the node controller where we were telling users what was being found.